src/components/NavigationBar/index.js

```diff
@@ -0,0 +1,28 @@
+import { h, Component } from 'preact'
+import style from './style.css'
+import classNames from 'classnames'
+import {preventDefaultOnClick} from '../utils'
+
+class NavigationBar extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {hover: false}
+  }
+
+  setHoverState = (state) => {
+    this.setState({hover: state})
+  }
+
+  render = ({back}) =>
+    <div className={style.navigation}>
+      <button href='#' className={classNames(style.back,{[style.backHover]: this.state.hover} )}
```

Comment:
what's this syntax? `{[style.backHover]: this.state.hover}`

Reply:
This is means return `style.backHover` if this.state.hover is `true`. https://github.com/JedWatson/classnames

Reply:
cool, do you need the square brackets?

Reply:
Yes, cause of the dot operator